docs/atlascli/command/atlas-projects-settings-update.txt

```diff
@@ -0,0 +1,103 @@
+.. _atlas-projects-settings-update:
+
+==============================
+atlas projects settings update
+==============================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Updates settings of a given project.
+
+Syntax
+------
+
+.. code-block::
+
+   atlas projects settings update [options]
+
+Options
+-------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - --disableCollectDatabaseSpecificsStatistics
```

Comment:
Question: Should this be without the "s" at the end of "specific"?
```suggestion
   * - --disableCollectDatabaseSpecificStatistics
```

Reply:
@sarahsimpers In the Scope & Spec docs it's referred as `--disableCollectDatabaseSpecificsStatistics`.

Reply:
+1 and in the endpoint doc as well https://www.mongodb.com/docs/atlas/reference/api/project-settings-get-one/